### DISCUSSION TITLE
[Scam Alert] PCoin پوماکوین

### DISCUSSION THREAD

**Discussion by Webamoozcom** [Webamoozcom/warning-list#35] [Announcements]
<div dir="rtl">

**Project Name:** 
PCoin

**نام پروژه:** 
پوماکوین 

**نماد پروژه:**
PCoin

**اتفاقی که افتاده است:**

باسلام

شرکت امپریال نتورک یه کمپانی جدید زده به نام

PCoin

یه ارز دیجیتال ساختن یه کوین جدید به این اسم هیچ جا هم لیست نشده جز 2 تا صرافی داخلی ایرانی

میگن به این صورت که شما بیا حدقل 100 دلار معادل تقریبا 3 میلیون تومن بیشترم میشه خرید ولی حدقل 100 دلار

پی کوین از ما بخر ما ماهانه 3 درصد معادل سالانه 36 درصد یعنی خیلی بیشتر از بانک بهت سود میدیم پولت پیش ما یکسال قفل میشه و نمیتونی دستش بزنی

این از این بخشش قسمت رفرال و زیرشاخه گیری بابت هر نفر 10 درصد از شارژی که اون میخره رو بهت میدیم مثال صد میلیون اگر پی کوین خرید 10 تومنش مال شماست

و خودشون رو نشون میدن که ماهانه 50 تا 100 میلیون پول میاد تو حسابشون و بهت میگن که تو هم میتونی و بیا شروع کن این اصلا نه جنس فروشی داره نه هیچی به سادگی میتونی پول در بیاری

شرکت پوماکوین متعلق به شرکت امپریال

![photo_2021-09-11_08-15-08](https://user-images.githubusercontent.com/87842536/132935346-1b27dba9-5bc9-4cbe-b5e0-fb22d0861cf5.jpg)

**قوانینی که نقض شده است:**

**نوع هشدار:**
هشدار طرح پانزی

**نوع کلاهبرداری:** 
طرح پانزی

**وضعیت کلاهبرداری:**
در جریان

**نام کلاهبرداران:**

**نام شرکت(ها):**
شرکت امپریال نتورک

**آدرس وب سایت(ها):**
https://www.pcoinglobal.com/
https://www.instagram.com/pumaa_coin/
https://www.instagram.com/pcoin_currency/
یکی از تیمهایی که با سیستم نتورک مشغول فعالیت هستن
https://instagram.com/pcoin_abolfazl?utm_medium=copy_link

**برآورد مبلغ کلاهبرداری:**

**منابع:**

**جمع بندی:**

**Comment by saeedkousha** [comment]
باسلام


شرکت امپریال نتورک یه کمپانی جدید زده به نام

 PCoin
 یه ارز دیجیتال ساختن یه کوین جدید به این اسم هیچ جا هم لیست نشده جز 2 تا صرافی داخلی ایرانی

میگن به این صورت که شما بیا حدقل 100 دلار معادل تقریبا 3 میلیون تومن بیشترم میشه خرید ولی حدقل 100 دلار

پی کوین از ما بخر ما ماهانه 3 درصد معادل سالانه 36 درصد یعنی خیلی بیشتر از بانک بهت سود میدیم پولت پیش ما یکسال قفل میشه و نمیتونی دستش بزنی

این از این بخشش قسمت رفرال و زیرشاخه گیری بابت هر نفر 10 درصد از شارژی که اون میخره رو بهت میدیم مثال صد میلیون اگر پی کوین خرید 10 تومنش مال شماست

و خودشون رو نشون میدن که ماهانه 50 تا 100 میلیون پول میاد تو حسابشون و بهت میگن که تو هم میتونی و بیا شروع  کن این اصلا نه جنس فروشی داره نه هیچی به سادگی میتونی پول در بیاری

**Reply by saeedkousha** [reply]
https://instagram.com/pcoin_abolfazl?utm_medium=copy_link
یکی از تیمهایی که با سیستم نتورک مشغول فعالیت هستن

**Reply by Webamoozcom** [reply]
سایتهاش و صرافی هایی که داخل ایران لیستش کرده اند را هم اگر بفرمایید ممنون میشم

**Reply by amir8037** [reply]
آدرس جدید وبسایت پوما
PcoCrypto.com

**Reply by nimashah1995** [reply]
اقامیلادی ک با اسم فیک اینجا اطلاعات میزاری یا اقای جرجندی ک از فرط بیکاری و بی اطلاعاتی اینجا دیتا میزارین و مردم رو میکشونید سمت شرکت های مزخرفی مثل لاکچری نتورک
ی کنفرانسی بزار تصویری تا هم خودتو و هم مردمو ک اینجا فریبشون میدی چون لیست افرادت رو تموم سیستم هایی پانزی دهه۸۰ ب باد فنا دادی بزار ی کنفرانس تا بهت بگم  و اگاهت کنم و مردمی ک اینجا کامنت گذاشتن و داری عوام فریبی میکنی رو هم حتما دعوت کن
مطمئنم نمیکنی اینکارارو

هرکی اینجا کامنت میزاره اگه واقعا دنبال اصل ماجراست درخواست بده ازین وبلاگ مزخرف تا ی جلسه تصویری بزاره هم باشن ببینم اصن میدونه بلاک چین و ارز چیه؟! و همه پروژه ها و .. ثابت بشه

**Reply by ahmadyazdanfard** [reply]
میشه یه پیام بهم بدی، شمارم09394599371

**Comment by MahdiyarJazmi** [comment]
ببینید فقط دارین چرت میگین چون از بانک سود بیشتری میده و تیم تریدر داره میشه کلاهبردار ؟؟؟
واقعا ک جهان سومی هستین و عقلتون عقبه

**Reply by amir8037** [reply]
عزیز همه مالباختگانی که اسیر این جور کلاهبرداریا میشن تعصبی برخورد میکنن،  طمع کورشون میکنه  ربط به شعور داره نه جهان اولی و سومی

**Reply by MahdiyarJazmi** [reply]
عزیز خب اخه اطلاعاتی ندارن
بعد یسری افرادم از روی سوزش میان چرت مینویسن و چون داره بلاکچین میاره بالا 
اطلاعات ندارن لال بمونن بهتره

**Reply by Mohammadsahbayy** [reply]
سلام من هم اوایل گول این حرفهاشونو خوردم و عضو شدم زمانی ک من وارد شدم قیمت pcoin یا همون پوما حدودا 8000تومان بود ک تقریبا باید 500 سکه میخریدی تا بتونی عضو فعالان بازارش بشی بعدم بری دنبال جذب سرمایه گذار تا سود نصیبت بشه اون موقع روی بلاکچین ترون کار میکردند و فقط در یک صرافی ایرانی لیست شده بود بنام نوین تکس ک تقریبا از لحاظ اسمی مشابهت با نوبیتکس داره ک خیلیا از این طریق گول میخورن خلاصه چن وقتی پیش سرمایه ام را خارج نمودم و حسابمو بستم و الانا شنیدم داره بلاکچین خودشو میزنه و از روی بلاکچین ترون میاد بیرون تا بتونه روی بلاکچین خودش قرار بگیره و اسمش جزو اسامی لیست انتظار بایننس دکس هست ک البته من هرچی لیست بایننس دکس رو بالا پایین کردم چیزی ندیدم و همچنان در همون صرافی نوین تکس الان با قیمت تقریبا 11000ت خریدو فروش میشه  الان چن سوال پیش روی منه
1. ارزی اگر لیست نشه مگر میتونه خودش قیمت بده
2.ارزی که لیست نباشه غیر از ایردرآپ چطوری میتونه خریدو فروش بشه اونم اگر بخوای اول خرید کنی و در طرح پانزی شرکت داده بشی باید حتما 500 سکه رو بخری
3. آیا ارزی میشه ک خود ارز خارجی باشه و مال استرالیا باشه اما در صرافی ایرانی لیست بشه  اونم یک صرافی ک نام و نشان مطرحی نداره یعنی میخوام بدونم ارزی هست ک لیست شدنش از ایران باشه
4. برای لیست شدن آیا فقط یک صرافی طبیعیه نباید حداقل در چن صرافی لیست باشه
5. این صرافی نوین تکس قیمت این ارز  رو چطور برآورد میکنه که با زمین خوردن معروف ترین ارزها و ریزش شدید کنونی  بازار ک هیچ ارزی از لحاظ قیمتی در جایگاه خودش بسر نمیبره و پایین تر قرار داره این ارز به سمت بالا حرکت میکنه و اصلا ریزشی رو متحمل نشده و همچنان صعودی عمل میکنه یعنی نشده ک بخواد نزولی بره الانم از 8000ت رسیده ب 11000ت قیمت این ارز رو این صرافی چطور برآورد میکنه
6. ودرآخر میخواستم بدونم هر ارزی ک در صرافی باشه مگر غیر از اینکه اون ارز ثبت شده و لیست شده آیا کوین مارکت کپ نباید جز 5000ارزی که داره اون رو نشون بده یا نه؟
ممنون میشم راهنماییم کنید و بگید چطور میشه یا بکجا میشه اطلاع داد تا در مورد این ارز که در ایران مشکوک ب کلاهبرداری هست تا مقامات پیگیری کنند ک بیشتر از این مالباخته بهش اضافه نشه ممنون

**Reply by nimashah1995** [reply]
من درخدمت بقیه سوالاتتون هستم

**Comment by Mohammadsahbayy** [comment]
سلام من هم اوایل گول این حرفهاشونو خورد و عضو شدم وقتی وارد شدم قیمت pcoin یا همون پوما حدودا 8000تومان بود تقریباً 500 سکه میخریدی تا بتونی عضو فعالان بازارش بشی بعدم بری جذب سرمایه گذر تا سود نصیبت بشه اون موقعیت روی بلوکچین. ترون کار میکردند و فقط در یک صرافی ایرانی لیست شده بود بنام نوین تکس که از نظر اسمی مشابه با نوبیتکس داره خیلیا از این طریق گول میخورن خلاصه چن وقتی پیش سرمایه ام را خارج کرد و حسابمو بستم و الان شنیدم داره بلوکچین خودشو میزنه. و از روی بلوکچین ترون میاد بیرون تا بتونه روی بلاکچین خودش قرار بگیره و اسمش جزو اسامی لیست بایننس دکس هست انتظار من هرچی لیست بایننس دکس رو بالا پایین کردم چیزی ندیدم و همچنان در همون صرافی نوین تکس الان با قیمت تقریبا 11000ت خریدو فروش. میشه الان چن سوال پیش روی منه

ارزی اگر لیست نشه مگر میتونه خودش قیمت بده
2.ارزی که لیست نباشه غیر از ایردرآپ چطوری میتونه خریدو فروش بشه اونم اگه بخوای اول خرید کنی و در طرح پانزی شرکت داد بشی باید حتما 500 سکه رو بخری.
آیا ارزی می شود که خود خارجی باشد و مال استرالیا باشد، اما در صرافی ایرانی لیست شود اونم یک صرافی ک نام و نشان مطرحی ندارد یعنی میخواهم بدونم ارزی هست که لیستش از ایران باشد.
برای لیست شدن آیا فقط یک صرافی طبیعیه نباید در چن صرافی لیست باشه
این صرافی نوین تکس قیمت این آرزو را چگونه می کند که با زمین خوردن معروف ترین ارزها و ریزش های شدید بازار ک هیچ ارزی از نظر قیمتی در جایگاه خود بسر نمیبره و پایین تر قرار دارد این آرزو به سمت بالا حرکت می کند و اصلا ریزشی رو می کند. متحمل نشده و ادامه روندی عمل می کند یعنی نمی خواهد نزولی بره الانم از 8000ت رسیده ب 11000ت قیمت این صرافی چگونه ارزیابی می کند
ودرآخر میخواستم بدونم هر ارزی ک در صرافی باشه مگر غیر از اینکه اون ارز ثبت شده و لیست شده آیا کوین مارکت کپ نباید جز 5000ارزی که داره رو اون نشون بده یا نه؟
ممنون میشم راهنماییم کنید و بگید چطور میشه یا بکجا میشه داد تا در مورد این ارز که در ایران مشکوک ب کلاهبرداری هست تا مقامات پیگیری کن بیشتر از این مالباخته بهش اضافه نشه ممنون

**Reply by nimashah1995** [reply]
شما بعد یکسال پولتو برداشت کردی؟

**Reply by nimashah1995** [reply]
طبق گفته خودت میگی پولتو برداشت کردی بعد از یکسال،و قطعا سود هاتم همشو گرفتی،الان مشکلت چیه ک شک داری؟!
اگه شک داری خب پولتو ببر باش بیت کوین بخر ک بقول خودت بهترین ارز دنیاس درصورتی ک bnb و Eth بهترین ارزای دنیان و همچنین سولانو، نه ارزی مثه بیت کوین ک امنیت و سرعتش پایینه و تو پلتفرمpow است، اما ب هرحال بگذریم،اگه شک داری به پروژه پوماکوین خب نخر عزیزدلم،برو بیت کوین بخر مثه هزاران نفری ک گول این تریدرارو خوردن و جو گیر شدن مثه بورس و رفتن تو قله خریدن و بازیچه دست نهنگ های بیت کوین شدن و زندگیشون بر باد رفت دقیق عین مالباخته عقده ای که این سایت رو از روی بیکاری و الافی زده چون خودش گول خورده و ی ادم شکست خورده بدبخته ک هیچ کار مثبتی نمیتونه بکنه،همه افراد شکست خورده ان ک فقط منفی هستن و شکاک و هیچکاری ک برای تکامل جهان هستی نمیتونن بکنن هیچ فقط وجودشون باعث میشه تکنولوژی عقب بیوفته،پروژه پوماکوین پروژه ایه ک هرکس توان تحمل و صبوری چند ساله و بینش چند ساله داره باید بخره تا دراینده مثل bnb رشد بزرگی بکنه چون ایده های جدیدی داره و در راستاش تلاش کرده و میکنه
این اطلاعاتی ک این سایت داده کاملا اشتباه و ناقص
دوست داشتین کامل براتون توضیح میدم
درضمن تو همین سایت گیت هاب تمام کدها و برنامه نویسی های پی کوین وجود داره و قابل سرچ است البته تو سایت گیت هاب اصلی نه تو ی وبلاگی ک ی ادم شکست خورده بی فایده اطلاعاتی اشتباهی رو فقط بالا آورده
شما تو سایت گیت هاب اصلی تمام دیتا هارو میتونی ریز ب ریز ببینی ک یک پروژه واقعی است و غیرقابل هک
درضمن لازم ب ذکره ک من پی کوین رو انتخاب کردم چون پلتفرمش ب نحوی طراحی شده ک نمیتونه نهنگ داشته باشه
و هیچوقت رشد شارپی نخواهد داشت پس ریزشی شارپی هم نخواهد داشت
از طرفی این پروژه زمین تا اسمون فرق میکنه با پروژه هایی ک این شخص تو این سایت نوشته
شما فقط کافیه رجوع کنید ب شخصی ک دیتای کافی و لازم ازین سیستم داره
و نکته مهم دیگه اینکه این پروژه چون جامعه استیکر ها و… رو داره ک اینجا نمیتونم بازش کنم چون شخصی ک این سایتو زده دنبال ایده هاس ک سریع بره کپی کنه یا مردمو جمع کنه ببره نیوشا و گیاهانی ک شما ی گونی شو میتونی از عطاری بخری ۱۰هزارتومن بیاد تو پک های زرقو برق بی فایده یک صدمش رو بفروشه ۲۰۰هزار تومن

**Reply by nimashah1995** [reply]
الانم هرچی من دیتا بزارم صاحب سایت چون جوابی نداره میاد پاک میکنه

**Reply by OMid7313** [reply]
پولتو با رشدش برداشتی سوداتم گرفتی بعد گول خوردی کاش همه همو اینجوری گول بزنن 😂

**Reply by Rezamuhammadi** [reply]
> طبق گفته خودت میگی پولتو برداشت کردی بعد از یکسال،و قطعا سود هاتم همشو گرفتی،الان مشکلت چیه ک شک داری؟! اگه شک داری خب پولتو ببر باش بیت کوین بخر ک بقول خودت بهترین ارز دنیاس درصورتی ک bnb و Eth بهترین ارزای دنیان و همچنین سولانو، نه ارزی مثه بیت کوین ک امنیت و سرعتش پایینه و تو پلتفرمpow است، اما ب هرحال بگذریم،اگه شک داری به پروژه پوماکوین خب نخر عزیزدلم،برو بیت کوین بخر مثه هزاران نفری ک گول این تریدرارو خوردن و جو گیر شدن مثه بورس و رفتن تو قله خریدن و بازیچه دست نهنگ های بیت کوین شدن و زندگیشون بر باد رفت دقیق عین مالباخته عقده ای که این سایت رو از روی بیکاری و الافی زده چون خودش گول خورده و ی ادم شکست خورده بدبخته ک هیچ کار مثبتی نمیتونه بکنه،همه افراد شکست خورده ان ک فقط منفی هستن و شکاک و هیچکاری ک برای تکامل جهان هستی نمیتونن بکنن هیچ فقط وجودشون باعث میشه تکنولوژی عقب بیوفته،پروژه پوماکوین پروژه ایه ک هرکس توان تحمل و صبوری چند ساله و بینش چند ساله داره باید بخره تا دراینده مثل bnb رشد بزرگی بکنه چون ایده های جدیدی داره و در راستاش تلاش کرده و میکنه این اطلاعاتی ک این سایت داده کاملا اشتباه و ناقص دوست داشتین کامل براتون توضیح میدم درضمن تو همین سایت گیت هاب تمام کدها و برنامه نویسی های پی کوین وجود داره و قابل سرچ است البته تو سایت گیت هاب اصلی نه تو ی وبلاگی ک ی ادم شکست خورده بی فایده اطلاعاتی اشتباهی رو فقط بالا آورده شما تو سایت گیت هاب اصلی تمام دیتا هارو میتونی ریز ب ریز ببینی ک یک پروژه واقعی است و غیرقابل هک درضمن لازم ب ذکره ک من پی کوین رو انتخاب کردم چون پلتفرمش ب نحوی طراحی شده ک نمیتونه نهنگ داشته باشه و هیچوقت رشد شارپی نخواهد داشت پس ریزشی شارپی هم نخواهد داشت از طرفی این پروژه زمین تا اسمون فرق میکنه با پروژه هایی ک این شخص تو این سایت نوشته شما فقط کافیه رجوع کنید ب شخصی ک دیتای کافی و لازم ازین سیستم داره و نکته مهم دیگه اینکه این پروژه چون جامعه استیکر ها و… رو داره ک اینجا نمیتونم بازش کنم چون شخصی ک این سایتو زده دنبال ایده هاس ک سریع بره کپی کنه یا مردمو جمع کنه ببره نیوشا و گیاهانی ک شما ی گونی شو میتونی از عطاری بخری ۱۰هزارتومن بیاد تو پک های زرقو برق بی فایده یک صدمش رو بفروشه ۲۰۰هزار تومن

بزرگوار اینجا سایت اصلی گیت هابه!‌و اینجایی ام که الان توشی وبلاگ شخصی نیست یه صفحه ی بحثه (discussion) و کسی سایت نزده. pow پلتفرم نیست الگوریتم اجماعه که همین pow که بقول شما بدرد نمیخوره یه انقلاب بود. موقعی که دغدغه ی کسایی که تلاش میکردن پول دیجیتال بسازن دوبار خرج کردن یا کپی کردن بود این مشکل رو حل کرد و کل بازار مدیون وجودشه . یه زمانی ام با کامپیوتر خونگیت با ماین کردن یه بلاک ۵۰ تا پاداش میگرفتی و بقول خودت اگه صبر داشتی الان نهنگ بودی!   من کاری به پروژه ندارم ولی زمانی که شما همین که داری میگی غیر قابل هک !! معلومه زیاد با این حوزه آشنا نیستی پس لااقل بخاطر سرمایه ی خودت که قطعا برات مهمه تعصب رو کنار بذار

**Comment by MahdiyarJazmi** [comment]
من برداشت کردم
زورتون نرسیده الکی دروغ پشتش نیارید
معاوم نیست از کدوم سیستم کلاهبرداری اومدین و کار میکنین حالا اومدین پی کوین
رو حاشیه بسازید
جمع کنید مسخره بازیاتونو بابا
برید بچسبید ب بادران و یونیک و بم

**Reply by nimashah1995** [reply]
والا منم برداشت کردم واقعا هم راضیم خداخیرشون بده
معلوم نیس کدوم بدبختی از سر بیکاری این سایتو زده و چ هدفی داره

**Comment by MahdiyarJazmi** [comment]
ولش عزیز منم ی غلطی کردم تیمم این سایتو داد و اومدم ببینم چخبره
اومدم فقط چرت و پرت دیدم نوشتن
امثال اینا فقط بادران و یونیک و بم خوبه براشون

**Comment by micro73** [comment]
pcoin (puma coin global)

ادرس سایت : https://pcocrypto.com/.

هشدار کلاهبرداری و اسکم

توضیح سایت:
ما یک پلت فرم ساخته ایمبرای خرید و فروش سهام.
PCCoin Global یک پلت فرم برای آینده تامین مالی است که به بلاک چین جدید سهام قدرت می دهد
در حالی که راه‌حل‌های موجود تنها برای حل یک مشکل در یک زمان ارائه می‌شوند، تیم ما آماده است تا یک محصول ایمن، مفید و با استفاده آسان بر اساس بلاک‌چین خصوصی بسازد. این شامل یکپارچه‌سازی آسان پرداخت‌های ارز دیجیتال و حتی یک سیستم داوری دیجیتال خواهد بود.
در پایان، هدف ما این است که همه شرکت‌ها، کارمندان و دارایی‌های تجاری را در یک اکوسیستم بلاک چین یکپارچه ادغام کنیم، که تجارت را واقعاً کارآمد، شفاف و قابل اعتماد خواهد کرد.

بررسی های صورت گرفته ::

ابتدا اینکه تیم سازنده این پلتفرم کاملا نامشخص و هویتی ندارند

به محض ورود به سایت trend micro check هشدار scam page میدهد
![2022-08-19_11-49-48](https://user-images.githubusercontent.com/103103225/185570548-4060662e-7e4e-4ed3-b149-afd988fa13bf.jpg)

بررسی اسکم ادوایرز سایت و ترجمه ماشینی ::
صاحب وب سایت از سرویسی برای پنهان کردن هویت خود در WHOIS استفاده می کند
طبق گفته Tranco این سایت دارای رتبه الکسا پایینی است
این وب سایت ممکن است خدمات ارزهای دیجیتال پرخطر ارائه دهد
این فروشگاه اینترنتی روش های پرداخت ناشناس را ارائه می دهد.
این وب سایت به عنوان یک کلاهبرداری احتمالی در Scamadviser گزارش شده است
ما ارجاعاتی به تلاش‌های بازاریابی چندسطحی را شناسایی کردیم که اغلب کلاهبرداری هستند
این وب سایت عمدتاً نقدهای منفی دریافت کرده است
این وب سایت (بسیار) جوان است.
این وب سایت در 30 روز گذشته توسط DNSFilter به عنوان تهدید علامت گذاری شده است
![2022-08-19_12-11-24](https://user-images.githubusercontent.com/103103225/185570587-bbf3e682-c1c8-4e3b-bf83-e01f07b7e5a5.jpg)
![2022-08-19_12-12-41](https://user-images.githubusercontent.com/103103225/185570591-3ff3b125-9389-4da1-9f00-e2c5788ab614.jpg)


اطلاعات whois سایت 👍
اطلاعات کاملا پروتکت شده هستند و از پرووایدری استفاده میکنند که در بررسی های قبلی مشخص شد اکثر اسکمر ها و کلاهبرداران از این پرووایدر استفاده میکنند
![2022-08-19_11-52-41](https://user-images.githubusercontent.com/103103225/185570620-44e06a95-0435-49c7-bbce-bce0acc46cc5.jpg)

شبکه های اجتماعی این سایت توییتر و تلگرام و اینستاگرام کاملا جدید و بدون کاربر و بدون اطلاعات هستند
![2022-08-19_11-56-23](https://user-images.githubusercontent.com/103103225/185570674-0e190c3a-9149-4669-9d93-7c2a9c44c982.jpg)
![2022-08-19_11-56-40](https://user-images.githubusercontent.com/103103225/185570678-70b92dc7-6bd6-4242-b9a4-f52ae0a37f72.jpg)
![2022-08-19_12-07-27](https://user-images.githubusercontent.com/103103225/185570693-3ce72bd1-391d-4771-96e2-6764883fa611.jpg)
![2022-08-19_12-07-58](https://user-images.githubusercontent.com/103103225/185570701-48e9b309-698b-4f15-9f7e-818741da6ffc.jpg)


ورودی های سایت تماما کاربران ایرانی هستند

![2022-08-19_11-54-19](https://user-images.githubusercontent.com/103103225/185570727-d75fd7ee-26fe-4891-a6ab-6b68423f5d48.jpg)

**Comment by OMid7313** [comment]
راه نجات مردم ایران pcoin ❤️💪👑
به جای اینکه از روی حسادتت اینکارو بکنی وقت میزاشتی چند نفرو به درآمد برسونی کمک کنی به مردم چهار نفر دعات کنن نه ۶۰۰۰ نفری که تا این لحظه پی کوین خریدن فوشت بدن که از این چند هزار نفر تاحالا یکی ناراضی نبوده این چند هزار نفر عقل ندارن تو داری 😉

**Comment by OMid7313** [comment]
خوبیش اینه وجودشو نداری بیای لایو زنده تا دستتو رو کنیم که دیگه نیای دروغ و چرتو پرت تحویل مردم بدی بدبخت فکر اخرتتم باش

**Reply by ahmadyazdanfard** [reply]
شمارمو میزارم یکیتون جواب منا بده، 9394599371

**Comment by Sufia66** [comment]
نمیدونم بر چه اساسی اینو میگی ، ولی قطعا باور فقر تو ذهن شما و امثال شما بیداد میکنه، به خاطر همینه که فقط ۳ درصد از مردم جهان فراوانی رو می‌فهمن و جذب میکنن، ذهن قدرتمند با باور فراوانی،  میگه خدای‌  بی نهایت و نامحدود چیز محدود خلق نمیکنه، تو که میگی سودهایی که میده بیشتر از سود بانک به خاطر اینکه در چارچوب ضوابط و شرایط محصور شدی و توان خروج از این ماتریکس رو نداری و به اشتباه میای از عقده ها و شکست هات مایه میذاری و هر وجودی رو نهی میکنی و هر  بودی رو منکر میشی ، نه تنها پروژه پی کوین بلکه هیچ وقت ذهن ثروت ساز نداری که همچین پروژه هایی رو ازش منتفع بشی . اونایی که بودن ، دیدن، موندن ، سود کردن، صبور بودن، عشق داشتن، سیستمو ار خودشون دونستن ، تلاش کردن، رشدش دادن، برای یک عمر کافین. من از کسانی هستم که از امثال شما و سایت های فیکتون که شخصیت پشتش نامعلومی ضربه خوردم، کاش هیچ وقت آگاهی اشتباهی که شما دادین رو باور نمیکردم ، کاش منافعتون رو میفهمیدم، با گفتن پانزی و شبکه و به خیال اینکه یه چیزی میفهمین مثل بقیه رشد نکردم تو این سیستم، در صورتی که این سیستم خدا بود ، این سیستم خیلی غیر ممکن هارو ممکن کرد و من شاهد همه اینها بودم. امیدوارم کارما کارتون بهتون برگرده که خیلیا مثل من به خاطر اراجیف شما ، سمت یه سرمایه گذاری امن نمیرن و سیستم بانکداری که کم کلاهبرداری نکرده از ملت رو چراغ راهم بدونم. شما در برابر همه کسانی که این سایت رو میبنن مسئولید. و به لطف خدا و خانواده ای که پشت این ارز هست همه کسانی که نیومدن روزی حسرت داشتن ۱۰ تا از این ارز رو می‌خورند. و اونوقت بشین هشدار کلاهبرداری بده. و امیدوارم همه کسانی که نشانه های پی کوین رو دریافت کردن و برای خرید از امثال شما مشورت گرفتن و مانع سود و رسیدن به درآمد عالی شدید ، کارماش به زندگیتون برگرده . این جهان کوه است و فعل ما ندا، سوی ما آید ندا ها را صدا. پژواک صدات به خودت بر میگرده دوست عزیز.

**Reply by micro73** [reply]
امیدوارم با دید باز و فکر این متن رو نوشته باشی...
اگر آگاهی و دانش این کار رو داشتی و یک گوشه بررسی کرده بودی این متن رو اینگونه نمینوشتی🙏
